blint/cli.py

```diff
@@ -195,6 +212,8 @@ def main():
     """Main function of the blint tool"""
     blint_options = handle_args()
 
+    blintdb_setup(blint_options)
```

Comment:
@aryan-rajoria We should not run blintdb_setup, which downloads blintdb, every single time someone runs blint with blintdb enabled. Some users will be on a closed network and as is, blintdb_setup will error out instead of using their offline copy of blintdb.

Reply:
with overwrite set as `false`. It only downloads the database if it is not already present. 
Please` suggest if I should still change this, I could make a condition to check if the database is already present?

Reply:
@aryan-rajoria Does "if it is not already present" mean any version may be present, or is it doing something like checking the hash and will only skip redownloading the same file? I don't know how you have the db named, but no overwrite would usually imply it just wouldn't redownload a file of either the same name or the same hash. 

Regardless, I need a scenario where the db is not present and for whatever reason cannot be downloaded successfully to be planned for so that the exception is handled gracefully rather than terminating in an unhandled exception.

Reply:
I see, got it!